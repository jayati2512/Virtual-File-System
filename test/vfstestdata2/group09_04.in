createvfs COUNTRY 1024
unmountvfs COUNTRY
mountvfs COUNTRY
makedir / state
makedir /state capital
makedir / union_territory
makedir /state/capital governor
makedir /state/capital/governor chiefminister
makedir / village
unmountvfs COUNTRY
createvfs TEMPLE 1024
mountvfs TEMPLE
makedir / places
makedir /places famous_place 
movedir /places/famous_place/priest
deletedir /union_territory
makedir / watch
makedir /watch brand
makedir /watch/brand price
unmountvfs TEMPLE
