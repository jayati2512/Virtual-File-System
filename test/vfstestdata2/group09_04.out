createvfs_SUCCESS
unmountvfs_FAILURE "ACCESS_DENIED"
mountvfs_SUCCESS
makedir_SUCCESS
makedir_SUCCESS
makedir_SUCCESS
makedir_SUCCESS
makedir_SUCCESS
makedir_SUCCESS
unmountvfs_SUCCESS
createvfs_SUCCESS
mountvfs_SUCCESS
makedir_SUCCESS
makedir_SUCCESS
movedir_FAILURE "VFS_INSUFFICIENT_ARGUMENTS"
deletedir_FAILURE "CANNOT_FIND_SPECIFIED_PATH_OR_DIR"
makedir_SUCCESS
makedir_SUCCESS
makedir_SUCCESS
unmountvfs_SUCCESS
